studio/components/interfaces/Settings/Logs/Logs.DatePickers.tsx

```diff
@@ -13,9 +18,16 @@ interface Props {
 const DatePickers: React.FC<Props> = ({ to, from, onChange, helpers }) => {
   const defaultHelper = getDefaultHelper(helpers)
   const [helperValue, setHelperValue] = useState<string>(to || from ? '' : defaultHelper.text)
+
   const handleHelperChange = (newValue: string) => {
-    setHelperValue(newValue)
     const selectedHelper = helpers.find((h) => h.text === newValue)
+    const fromTime = selectedHelper?.calcFrom()
```

Comment:
I personally feel that this shouldn't be here. the state on whether to show the upgrade prompt is irrelevant to the datepicker, the datepicker only caches the selected time in state and triggers the callback on selection.